bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi.properties

```diff
@@ -1,18 +1,22 @@
 # binding
+
+binding.urtsi.name = Somfy URTSI II Binding
```

Comment:
I suggest a change in binding.xml to use the same approach for label and description, either both using `@text` or none.

Reply:
Done.